python/scripts/add_users_all_org_members_github_team.py

```diff
@@ -1,55 +1,50 @@
+"""
+
+"""
```

Comment:
Empty docstring? Is this to appease the pylint gods? 😺

Reply:
I may or may not have considered writing a docstring 🙈

Then decided that the comments next to the variables were better suited for what I was trying to articulate and completely forgot about this 😂